scania-digital-design-system/sdds
# fix(file-extension): file extensions of stories files.

**Describe pull-request**  
Changed file extensions and resolved typescript errors

**Solving issue**  
Fixes: [AB#2581](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2581) [AB#2582](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2582)

**How to test**  
1. Go to storybook link below. 
2. Check in Components.
3. Check that the stories still works as intended. 

**Screenshots**  
-

**Additional context**  
-
